emr-setup.sh

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env bash
+
+set -e
```

Comment:
you want these enabled after testing?

Reply:
Yeah its safer to exit at first error for future when someone works on this script

Reply:
Sorry, I meant all the `set -*` you have.  -x will be especially verbose when running the script.  IMO, There's no need for this when not testing.

Reply:
removed